contrib/bundle/Vagrantfile

```diff
@@ -0,0 +1,61 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "generic/ubuntu2204"
```

Comment:
hm can you expand on why we need to use vagrant if we're using ubuntu? aren't there native runners for ubuntu on gh actions

Reply:
I used it for being able to test it locally, the native builders have pre set environments (like pre-installed docker/podman) including configuration files.